tests/queries/0_stateless/02735_system_zookeeper_connection.reference

```diff
@@ -1,2 +1,2 @@
-default	::1	9181	0	0	3
-zookeeper2	::1	9181	0	0	0
+default	::1	9181	0	0	1	0	3
+zookeeper2	::1	9181	0	0	0	0	0
```

Comment:
@guoxiaolongzte, it means that `connected_time` was **not** between `yesterday()` and `today()` and the difference between `session_uptime_elapsed_seconds` and `zookeeperSessionUptime()` was more than 10 seconds. It means that the time in the table is still incorrect

Reply:
I feel that verifying this is meaningless. please see the result。/root/guoxiaolong/ClickHouse/build/programs/clickhouse-server --config /root/guoxiaolong/ClickHouse/programs/server/config.xml
start at 2023-06-19 23:26:24 in beijing time.
SELECT
    *,
    zookeeperSessionUptime()
FROM system.zookeeper_connection

Query id: 661f5959-54fa-411e-afea-15f2adb7dcce

Row 1:
──────
name:                           default
host:                           127.0.0.1
port:                           9181
index:                          0
connected_time:                 2023-06-19 23:26:24
session_uptime_elapsed_seconds: 2261
is_expired:                     0
keeper_api_version:             3
client_id:                      1
zookeeperSessionUptime():       2261

1 row in set. Elapsed: 0.033 sec.

Reply:
abs(session_uptime_elapsed_seconds - zookeeperSessionUptime())   always equal to 0，so abs(session_uptime_elapsed_seconds - zookeeperSessionUptime())  <10 always equal to 1.

connected_time always standard time to initiate connection zk.

Reply:
> so abs(session_uptime_elapsed_seconds - zookeeperSessionUptime()) <10 always equal to 1

Yes, so why did you replace 1 with 0 in the reference file? It should be 1.

Reply:
yes，so we delete (connected_time between yesterday() and today())     and  abs(session_uptime_elapsed_seconds - zookeeperSessionUptime()) <10 set 1?

Reply:
is the pr ok?@tavplubix